switchmap/server/db/ingest/update/zone.py

```
@@ -213,27 +212,28 @@ def ip(self):
-            # Get hostname for DB
-            if bool(dns) is True:
+        ips = [item.ip for item in self._arp_table]
+
+        unique_ips = set(ips)
+        hostname_map = {}
+        if not test:
+            for ip in unique_ips:
                 try:
-                    hostname = socket.gethostbyaddr(item.ip)[0]
+                    hostname_map[ip] = socket.gethostbyaddr(ip)[0]
                 except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)